_rules/SC1-2-video-transcript.md

```diff
@@ -24,15 +21,17 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming][] video element [visible in the page][]
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio.
 
 ### Expectation 1
 
-A [text transcript][] is available for the test target, either on the page or available through a link (or some other mechanism).
+A text transcript is available for the test target, either on the page or available through a link.
```

Comment:
"Text transcripts are available for applicable targets, ..." would be better than test targets (because these can be inapplicable)

Reply:
Similar to previous comment. Test targets are always the applicable elements. This is a term borrowed from the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/#output-test-target)